### ISSUE TITLE
Moderation workflow timeline steps should be distinguishable with other means than color

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8484]
### Issue Summary

In our moderation workflows for pages, we use a timeline component so users can tell what steps a page has to go through before being published. This timeline component is too dependent on color for users to understand which step the page is currently at:

![timeline-bw](https://user-images.githubusercontent.com/877585/166694161-339394cd-16cb-4798-a73b-ca522977044e.png)

Here, the page is at the "Moderators approval" step, after having been through the "Soumis" (Submitted) step.

### Steps to Reproduce

1. Submit a page for moderation
2. Go back to the page, in the Info side panel click "View details"
3. Look at the timeline component inside the modal

---

We need a solution that addresses two issues:

- For sighted users, we need a clear distinction. It could be argued the saturation of the colors is different enough to understand which step is the most prominent, and infer from this it’s the current step – but really it’d be better for there to be a clearer cue that’s completely independent of the contrast between the different steps.
- For blind users, we need a distinction that’s independent of the visuals. This could be screen-reader-only text, or the same solution as for sighted users.

This component seems to have used tooltips at some point, with clearer labels for steps. However I’ve never seen them work, and even if they did, we need a distinction that’s available even without relying on tooltips (hover-only, no keyboard support).

**Comment by mahamtariq58** [comment]
I would like try this issue !

**Comment by Scotchester** [comment]
@mahamtariq58 Go for it! We don't generally assign issues unless they are major core features, so you are welcome to take a stab at it.

**Comment by sdmoran** [comment]
@mahamtariq58 are you still working on this? If not I would like to give it a try :)

**Comment by Dev-Liz** [comment]
Hello @Scotchester i'd love to come up with a design to fix this issue. Can I give it a shot?

Also, i'd like to understand the "Step to Reproduce" better. I don't seem to grasp that part.

**Comment by gasman** [comment]
Hi @Dev-Liz! Sorry, this one's already taken - @sdmoran has submitted a proposed fix for this at #9305.
